packages/concerto-cto/lib/printer.js

```diff
@@ -147,6 +147,16 @@ function typeFromMetaModel(mm){
     return result;
 }
 
+/**
+ * Returns true if the typeof the object === 'undefined' or
+ * the object === null.
+ * @param {Object} obj - the object to be tested
+ * @returns {boolean} true if the object is null or undefined
+ */
+function isNull(obj) {
```

Comment:
Why not import  Util.isNull from concerto-core? E.g. we could move isNull into ModelUtil and get rid of this Util class.

Reply:
I was trying to avoid adding another dependency (and dependency loops). I'll see if there's some way to hoist this into `@accordproject/concerto-util`